src/seedu/addressbook/commands/EditCommand.java

```diff
@@ -0,0 +1,77 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.person.UniquePersonList.DuplicatePersonException;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+import seedu.addressbook.data.tag.Tag;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ *Edit the particulars of a contact in the addressBook
+ */
+public class EditCommand extends Command{
+    public static final String COMMAND_WORD = "edit";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits a contact in the address book. "
+            + "Enter the index of the contact to be edited,followed by a '/ '.\n"
+            + "If the contact exists, enter the new particulars of this contact in the correct format.\n"
+            + "Contact details can be marked private by prepending 'p' to the prefix.\n"
+            + "Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...\n"
+            + "Example: " + COMMAND_WORD
+            + " John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney";
+
+    public static final String MESSAGE_SUCCESS = "Contact is edited: %1$s";
+    //public static final String MESSAGE_NONEXISTENT_PERSON = "This person does not exist in the address book";
```

Comment:
Eliminate unused code and do not retain them commented, you can always get it back from previous versions if necessary.